app/src/main/java/com/vultisig/wallet/models/Coin.kt

```diff
@@ -47,7 +47,7 @@ object Coins {
             address = "",
             decimal = 8,
             hexPublicKey = "",
-            priceProviderID = "bitcoin-cash",
+            priceProviderID = "Bitcoin Cash",
```

Comment:
we're using this field for accessing prices, isn't that breaking that?

Reply:
No, When I chanaged this field, the output changed from following
![Screenshot 2024-05-25 at 10 51 01](https://github.com/vultisig/vultisig-android/assets/32006742/d8a65450-803b-4e74-9804-2b29fe37cf35)

to the following

![Screenshot 2024-05-25 at 10 53 06](https://github.com/vultisig/vultisig-android/assets/32006742/9522f9cf-beda-412c-a75d-f8710083dc95)

Reply:
Fixed, please check again